packages/jsxstyle-utils/src/getStyleCache.ts

```diff
@@ -2,6 +2,26 @@ import getStyleKeysForProps from './getStyleKeysForProps';
 import addStyleToHead from './addStyleToHead';
 import stringHash from './stringHash';
 
+export interface InsertRuleCallback {
+  (rule: string, props?: {}): boolean | void;
+}
+
+export interface StyleCache {
+  reset: { (): void };
```

Comment:
Is this the same as `reset(): void`?

Reply:
@steadicat heh, yeah. can you tell i’m new to TypeScript? 😅